Fix cross-compil on windows

```console
# github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall
[goxc:xc] 2015/11/18 15:55:43 'go' returned error: Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall
[goxc:xc] 2015/11/18 15:55:43 Error: Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall
[goxc:xc] 2015/11/18 15:55:43 Have you run `goxc -t` for this platform (386,windows)???
# github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall
[goxc:xc] 2015/11/18 15:55:56 'go' returned error: Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall
[goxc:xc] 2015/11/18 15:55:56 Error: Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall
[goxc:xc] 2015/11/18 15:55:56 Have you run `goxc -t` for this platform (amd64,windows)???
[goxc:xc] 2015/11/18 15:56:21 Multiple errors (returning first one): [Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall]
[goxc:xc] 2015/11/18 15:56:21 Stopping after 'xc' failed with error 'Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall'
[goxc] 2015/11/18 15:56:21 RunTasks error: Wait error: exit status 2: # github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client
../../vendor/github.com/moul/gotty-client/gotty-client.go:246: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:247: undefined: syscall.SIGWINCH
../../vendor/github.com/moul/gotty-client/gotty-client.go:251: undefined: syscall.SYS_IOCTL
../../vendor/github.com/moul/gotty-client/gotty-client.go:252: undefined: syscall.TIOCGWINSZ
../../vendor/github.com/moul/gotty-client/gotty-client.go:253: not enough arguments in call to syscall.Syscall
make: *** [goxc] Error 1
```

--- a/.goxc.json
+++ b/.goxc.json
@@ -1,6 +1,7 @@
 {
 	"AppName": "gotty-client",
 	"ArtifactsDest": "dist",
+	"BuildConstraints": "!plan9,!nacl,!solaris",
 	"OutPath": "{{.Dest}}{{.PS}}{{.Version}}{{.PS}}{{.ExeName}}_{{.Version}}_{{.Os}}_{{.Arch}}{{.Ext}}",
 	"TasksExclude": [
 		"go-test",

--- a/README.md
+++ b/README.md
@@ -64,7 +64,7 @@ USAGE:
    gotty-client [global options] command [command options] GOTTY_URL
 
 VERSION:
-   1.1.0
+   1.3.0+
 
 AUTHOR(S):
    Manfred Touron <https://github.com/moul/gotty-client>
@@ -73,6 +73,7 @@ COMMANDS:
    help, h	Shows a list of commands or help for one command
 
 GLOBAL OPTIONS:
+   --debug, -D		Enable debug mode [$GOTTY_CLIENT_DEBUG]
    --help, -h		show help
    --version, -v	print the version
 ```
@@ -103,6 +104,7 @@ $ brew install https://raw.githubusercontent.com/moul/ssh2docker/master/contrib/
 
 ### master (unreleased)
 
+* Remove solaris,plan9,nacl for `.goxc.json`
 * Add an error if the go version is lower than 1.5
 * Flexible parsing of the input URL
 * Add tests

--- a/arch.go
+++ b/arch.go
@@ -1,0 +1,34 @@
+// +build !windows
+
+package gottyclient
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"unsafe"
+)
+
+func notifySignalSIGWINCH(c chan<- os.Signal) {
+	signal.Notify(c, syscall.SIGWINCH)
+}
+
+func resetSignalSIGWINCH() {
+	signal.Reset(syscall.SIGWINCH)
+}
+
+func syscallTIOCGWINSZ() ([]byte, error) {
+	ws := winsize{}
+
+	syscall.Syscall(syscall.SYS_IOCTL,
+		uintptr(0), uintptr(syscall.TIOCGWINSZ),
+		uintptr(unsafe.Pointer(&ws)))
+
+	b, err := json.Marshal(ws)
+	if err != nil {
+		return nil, fmt.Errorf("json.Marshal error: %v", err)
+	}
+	return b, err
+}

--- a/arch_windows.go
+++ b/arch_windows.go
@@ -1,0 +1,16 @@
+package gottyclient
+
+import (
+	"errors"
+	"os"
+)
+
+func notifySignalSIGWINCH(c chan<- os.Signal) {
+}
+
+func resetSignalSIGWINCH() {
+}
+
+func syscallTIOCGWINSZ() ([]byte, error) {
+	return nil, errors.New("SIGWINCH isn't supported on this ARCH")
+}

--- a/gotty-client.go
+++ b/gotty-client.go
@@ -9,13 +9,10 @@ import (
 	"net/http"
 	"net/url"
 	"os"
-	"os/signal"
 	"regexp"
 	"strings"
 	"sync"
-	"syscall"
 	"time"
-	"unsafe"
 
 	"github.com/moul/gotty-client/vendor/github.com/Sirupsen/logrus"
 	"github.com/moul/gotty-client/vendor/github.com/creack/goselect"
@@ -243,23 +240,16 @@ type winsize struct {
 func (c *Client) termsizeLoop(wg *sync.WaitGroup) {
 	defer wg.Done()
 	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, syscall.SIGWINCH)
-	defer signal.Reset(syscall.SIGWINCH)
-	ws := winsize{}
+	notifySignalSIGWINCH(ch)
+	defer resetSignalSIGWINCH()
 
 	for {
-		syscall.Syscall(syscall.SYS_IOCTL,
-			uintptr(0), uintptr(syscall.TIOCGWINSZ),
-			uintptr(unsafe.Pointer(&ws)))
-
-		b, err := json.Marshal(ws)
-		if err != nil {
-			logrus.Warnf("json.Marshal error: %v", err)
-		}
-
-		err = c.write(append([]byte("2"), b...))
-		if err != nil {
-			logrus.Warnf("ws.WriteMessage failed: %v", err)
+		if b, err := syscallTIOCGWINSZ(); err != nil {
+			logrus.Warn(err)
+		} else {
+			if err = c.write(append([]byte("2"), b...)); err != nil {
+				logrus.Warnf("ws.WriteMessage failed: %v", err)
+			}
 		}
 		select {
 		case <-c.QuitChan:

